sql/npc_list.sql

```diff
@@ -10258,7 +10258,7 @@ INSERT INTO `npc_list` VALUES (17162680,'blank','     ',192,-50.000,-5.349,212.5
 INSERT INTO `npc_list` VALUES (17162681,'Myron_CA','Myron, C.A.',223,-13.534,-4.999,119.040,14,40,40,0,0,0,2,27,0x01000F0196103920963096409650006000700000,32,'WOTG',1);
 INSERT INTO `npc_list` VALUES (17162682,'Wenonah_CA','Wenonah, C.A.',194,-2.175,-1.000,10.184,8,40,40,0,0,0,0,27,0x0100070296103920963096409650006000700000,32,'WOTG',1);
 INSERT INTO `npc_list` VALUES (17162683,'Hampu-Kampu','Hampu-Kampu',245,-115.597,-1.000,-158.703,1,40,40,0,0,0,0,27,0x01000C0577107B20693069406950006000700000,32,'WOTG',1);
-INSERT INTO `npc_list` VALUES (17162684,'Door_Acolyte_hostel','Door:Acolyte hostel',0,124.000,-3.000,222.215,32769,40,40,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'WOTG',0);
+INSERT INTO `npc_list` VALUES (17162684,'Door_Acolyte_hostel_up','Door:Acolyte hostel',0,146.619,-8.525,242.874,32769,40,40,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'WOTG',0);
```

Comment:
I assume these doors were mixed up at some point, explaining the swap.

Reply:
Yeah actually those door were named exactly the same 😹 and one of the scripts was MIA 🙀